docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.4"
+
+services:
+  db:
+    image: postgres:11.5-alpine
+    tmpfs: /var/lib/postgresql/data
+    ports:
+      - "${PORT_DB-5432}:5432"
+    environment:
+      POSTGRES_DB: graphile_worker_test
+
+  app:
+    build:
+      context: .
+    volumes:
+      - ./__tests__:/app/__tests__
+      - ./sql:/app/sql
+      - ./src:/app/src
+      - ./perfTest:/app/perfTest
```

Comment:
Why are we copying these files in the Dockerfile if we're also mounting them here?

Reply:
They're copied so that the Dockerfile is usable without the docker-compose config - there's no reason you can't just run it directly with docker. But we also mount them here so that for local dev you don't have to rebuild the docker image every time you make a change. I can see how it's a bit redundant, but I think it'd be a bit odd to ship a Dockerfile that didn't work without specific directories mounted.

Reply:
Got it; thanks :+1: